packages/react/package.json

```
@@ -11,7 +11,7 @@
     "@stardust-ui/react-component-ref": "^0.39.0",
     "@stardust-ui/react-proptypes": "^0.39.0",
     "classnames": "^2.2.5",
-    "downshift": "~3.2.10",
+    "downshift": "3.2.14",
```

```suggestion
    "downshift": "~3.2.14", // to match `3.2.*`
```
_or_
```suggestion
    "downshift": "^3.2.14", // to match `3.*.*`
```

We should not require exact version of libraries, until it is completely broken in other versions. If there will be other library that requires `downshift@^3.2.15` or `downshift@^3.3.0` it will end with bloated bundle for consumers.